gadopt/approximations.py

```diff
@@ -89,6 +89,9 @@ def __init__(self, Ra, kappa=1, g=1, rho=1, alpha=1):
     def buoyancy(self, p, T):
         return self.Ra * self.g * self.alpha * self.rho * T
 
+    def rho_field(self, p, T):
+        return self.rho * (1 - self.alpha * T)
+
```

Comment:
done

Reply:
as you said here https://github.com/g-adopt/g-adopt/pull/51#pullrequestreview-1986455985 i've moved the definition of free surface density into approximation, as well as the flag to turn it on/off, so that it is clear that this density is associated with the free surface external stress term